analyzers/tests/SonarAnalyzer.UnitTest/TestCases/StringLiteralShouldNotBeDuplicated.CSharp11.cs

```diff
@@ -22,3 +22,21 @@ public class FooLessThanFiveCharacters
 
     public static readonly string NameReadonly = """foo""";
 }
+
+public class FooNonCompliantStringInterpolation
+{
+    static string GetName() => "Bar";
```

Comment:
Needs at least 5 characters:
```suggestion
    static string GetName() => "BarBar";
```